app/views/posts/show.html.erb

```diff
@@ -1 +1,25 @@
-<h1>Here you will find a list of posts for a given user</h1>
\ No newline at end of file
+<a href="<%= user_posts_path(@user) %>">
+  <button class="custom-back-btn">⬅</button>
+</a>
+<div class="custom-post-container">
+  <h2>Post <%= "#" + @post.id.to_s %> by <%= @user.name %></h2>
+  <div class='custom-post-interactions'>
+    <span>Comments: <%= @post.comments_counter %>,</span>
+    <span>Likes: <%= @post.likes_counter %></span>
+  </div>
+  <p><%= @post.text %></p>
+</div>
+<div class="custom-post-container">
+<h3>Comments:</h3>
+<% if @post.comments.present? %>
+  
+  <% @post.comments.each do |comment| %>
+    <div class="custom-post-comments">
+      <span><%= comment.author.name %>:</span>
```

Comment:
- [ ]Also there is an error when I try to access your post-show due to some misspelling,` comments_counter` should be `commentsCounter`,` likes_counter` should be l`ikesCounter` and it can not access `comment.author.name` kindly fix these issues